autoload/airline/themes/dracula.vim

```diff
@@ -7,96 +7,84 @@
-let s:N3 = [ s:guiWhite , s:gui01 , s:ctermWhite , s:cterm01 ]
-
-" Insert mode
-let s:I1 = [ s:guiBlack , s:gui07 , s:ctermBlack , s:cterm07 ]
-let s:I2 = [ s:guiWhite , s:gui02 , s:ctermWhite , s:cterm02 ]
-let s:I3 = [ s:guiWhite , s:gui01 , s:ctermWhite , s:cterm01 ]
-
-" Visual mode
-let s:V1 = [ s:guiBlack , s:gui06 , s:ctermBlack , s:cterm06 ]
-let s:V2 = [ s:guiWhite , s:gui02 , s:ctermWhite , s:cterm02 ]
-let s:V3 = [ s:guiWhite , s:gui01 , s:ctermWhite, s:cterm01 ]
-
-" Replace mode
-let s:R1 = [ s:guiBlack , s:gui05 , s:ctermWhite, s:cterm05 ]
-let s:R2 = [ s:guiWhite , s:gui02 , s:ctermWhite, s:cterm02 ]
-let s:R3 = [ s:guiWhite , s:gui01 , s:ctermWhite, s:cterm01 ]
-
-" File changed
-let s:changed = [ s:guiWhite , s:guiChangedColor , s:ctermWhite , s:ctermChangedColor ]
-
-let g:airline#themes#dracula#palette = {}
-let g:airline#themes#dracula#palette.normal = airline#themes#generate_color_map(s:N1, s:N2, s:N3)
-let g:airline#themes#dracula#palette.insert = airline#themes#generate_color_map(s:I1, s:I2, s:I3)
-let g:airline#themes#dracula#palette.visual = airline#themes#generate_color_map(s:V1, s:V2, s:V3)
-let g:airline#themes#dracula#palette.visual = airline#themes#generate_color_map(s:V1, s:V2, s:V3)
-let g:airline#themes#dracula#palette.replace = airline#themes#generate_color_map(s:R1, s:R2, s:R3)
-
-" Inactive mode
-let s:IN1 = [ s:gui04 , s:guiWhite , s:cterm04 , s:ctermWhite ]
-let s:IN2 = [ s:gui04 , s:gui01 , s:cterm04  , s:cterm01 ]
-let s:IA = [ s:IN1[1] , s:IN2[1] , s:IN1[3] , s:IN2[3] , '' ]
-let g:airline#themes#dracula#palette.inactive = airline#themes#generate_color_map(s:IA, s:IA, s:IA)
-
-" Warning info
-let s:WARNING = [ s:guiBlack, s:gui03, s:ctermBlack, s:cterm03 ]
-let s:ERROR = [ s:guiWhite, s:gui05, s:ctermWhite, s:cterm05 ]
-
-let g:airline#themes#dracula#palette.normal.airline_warning = s:WARNING
-let g:airline#themes#dracula#palette.insert.airline_warning = s:WARNING
-let g:airline#themes#dracula#palette.visual.airline_warning = s:WARNING
-let g:airline#themes#dracula#palette.replace.airline_warning = s:WARNING
-
-let g:airline#themes#dracula#palette.normal.airline_error = s:ERROR
-let g:airline#themes#dracula#palette.insert.airline_error = s:ERROR
-let g:airline#themes#dracula#palette.visual.airline_error = s:ERROR
-let g:airline#themes#dracula#palette.replace.airline_error = s:ERROR
-
-" File modified and not saved
-let g:airline#themes#dracula#palette.normal_modified = airline#themes#generate_color_map(s:N1, s:N2, s:changed)
-let g:airline#themes#dracula#palette.insert_modified = airline#themes#generate_color_map(s:I1, s:I2, s:changed)
-let g:airline#themes#dracula#palette.replace_modified = airline#themes#generate_color_map(s:R1, s:R2, s:changed)
-let g:airline#themes#dracula#palette.visual_modified = airline#themes#generate_color_map(s:V1, s:V2, s:changed)
-
-" CtrlP
-if !get(g:, 'loaded_ctrlp', 0)
-  finish
+"}}}
+" Helpers: {{{
+
+func! s:clr(fg, bg, ...)
```

Comment:
I would enjoy seeing some documentation (read: comments) on the usage of the script-local functions, here and elsewhere, since Vimscript isn't necessarily the easiest thing to read and we're dealing with a very specific data structure for airline. For future maintenance, it could be nice.